incident_response_checklist.md

```diff
@@ -11,6 +11,7 @@ For the engineer managing a production incident.
     - If it impacts many users:
         - <span style="color:darkorange">`4 hrs`</span> Daytime: Confirm that a support specialist is aware of the issue (slack #dsde-comms or @dspsupportteam in #workbench-resilience) and can put up a banner. Provide the Jira ticket and a description of the user impact.
         - <span style="color:darkorange">`8 hrs`</span> Off-hours: [Put up a banner in Terra.](https://fcprod-jenkins.dsp-techops.broadinstitute.org/job/terra-service-banner) (You will need to be on the VPN and authed in [prod jenkins](https://fcprod-jenkins.dsp-techops.broadinstitute.org/) with your github account)
+            - In order to set up the banner, from the Jenkins job, click on "Build With Parameters" and you'll see a screen with free text fields and information about default values for a banner. After filling in a title, message and any links for the banner click on "Build" and that should trigger a job that loads a banner with that information into the Terra UI.       
```

Comment:
(we could definitely do to make this clearer, so) per Eric in UX, we don't want to encourage folks to use their own verbage for the banner.  We'd rather have them stick to the default to avoid accidentally creating a bad UX.

I would reword this to "Use the defaults unless there is particular information you can convey about where the problem is, for example, "We are experiencing an issue with submitting workflows."  Put that information in the message.  Keep defaults for title and link."

Reply:
Also – I'm wondering if we can put this information in the Jenkins job itself rather than in the checklist, to avoid the checklist getting too long

Reply:
made it as minimal as I could by taking away any language on how to fill in the banner. FYI -- the Jenkins job indicates free text is allowed-- I'm not sure if you own the Jenkins job for the banner but it might be worth updating that language in Jenkins directly: 
![image (25)](https://user-images.githubusercontent.com/14941133/109655218-05737a00-7b31-11eb-86f9-80fcb04edfc1.png)